services/network/loadbalancer/wssd.go

```diff
@@ -236,8 +286,23 @@ func getWssdLoadBalancer(networkLB *network.LoadBalancer, group string) (wssdClo
 				if fipcf.IPAddress != nil {
 					wssdCloudLB.FrontendIP = *fipcf.IPAddress
 				}
+				if fipc[0].Name != nil {
+					wssdCloudLBFIpC.Name = *fipc[0].Name
```

Comment:
>wssdCloudLBFIpC [](http://example.com/codeflow?start=5&length=15)

i think we should not create a partial object meaning wssdCloudLBFIpC with only name but no public ip ref